gems/pending/util/win32/miq-powershell.rb

```diff
@@ -83,7 +83,11 @@ def self.verify_return_object(xml)
   end
 
   def self.is_error_object?(xml)
-    object_type = xml.root.elements[1].attributes['Type'] rescue ""
+    if MiqXml.rexml_parser?
+      object_type = xml.root.elements[1].attributes['Type'] rescue ""
+    else
+      object_type = xml.root.elements[0].attributes['Type'].value rescue ""
+    end
     return true if !object_type.nil? && object_type.split('.').last == 'ErrorRecord'
```

Comment:
maybe too much of a change / part of a separate PR / for another day far far away but...

```ruby
# before:
return true if !object_type.nil? && object_type.split('.').last == 'ErrorRecord'
return false

# after:
object_type && object_type.split('.').last == 'ErrorRecord'
```